src/main/java/com/singh/astha/notification/service/utils/Constants.java

```diff
@@ -44,6 +44,10 @@ public final class Constants {
 
     public static final String REQUEST_ARRIVAL_TIME = "requestArrivalTime";
 
+    public static final String SUCCESSFULLY_SENT_NOTIFICATION = "Successfully sent notification";
+
+    public static final String SUCCESSFULLY_ENQUEUED_MESSAGE_IN_KAFKA = "Successfully enqueued message in Kafka";
+
```

Comment:
Messages should be put into MessageConstants

Reply:
fixed